### DISCUSSION TITLE
Add a "Sort by Hot" or a "sort by active"

### DISCUSSION THREAD

**Discussion by LunaWurx** [modrinth/.github#8] [Feature requests]
### Describe the solution you'd like

Add a "Sort by hot" filter. It sorts by most follows and downloads in the past hour/minute/day. 

### Describe alternatives you've considered

none, really

### Additional context

This shows mods that are popular in current times and/or ones that are becoming popular. This helps new projects to be discovered easier and older mods that are used less to be less dominant

**Comment by DrParadox7** [comment]
Yeah, this is a feature Modrinth lacks that is detrimental to the ease of use on the platform.
Right now users have to scroll through the most downloaded projects and scrutinize which projects are good and which are just old while new projects don't really get seen.

While having an option to sort through the past hour/minute/day is nice, as long as Modrinth matches the functionality of CurseForge it would suffice. 

I believe they just check for daily downloads (Total Downloads/Project Age) but I could be wrong
![image](https://user-images.githubusercontent.com/47131096/183051354-9b84760f-3355-4c1d-83f8-eb12f9632280.png)

**Reply by RandomStringOfLetter** [reply]
personally I think curseforges curseforges popularity sorting is very vague as you don't really know what it is doing but I do agree on the rest.

**Comment by RandomStringOfLetter** [comment]
You would think the dev would've added this feature if they wanted to long ago, I think this feature is overlooked and underrated although it can be very helpful. I think the option should be customizble, so instead of just  a sort by "hot/active" button only you get a separate drop down like with the most downloaded (in other sites) filter where you could choose the time frame and/or game version.

**Comment by CrystalClear51** [comment]
why only stop at /day? There should also be /week and /month, i mean i watch mod reviews alot and just quickly going to the most popular mods of the month quickly instead of spending like 10-20mins watching a mod review would be very convenient

**Comment by RandomStringOfLetter** [comment]
Do they check the discussions?
